python/qpid_dispatch/management/qdrouter.json

```diff
@@ -841,6 +841,46 @@
                     "type": "string",
                     "description": "Where to send log messages. Can be 'stderr', 'stdout', 'syslog' or a file name.",
                     "update": true
+                },
+                "default": {
+                    "type": "integer",
+                    "description": "how many entries in the severity-level histogram at the DEFAULT level",
+                    "update": "true"
+                },
+                "trace": {
+                    "type": "integer",
+                    "description": "how many entries in the severity-level histogram at the TRACE level",
+                    "update": "true"
```

Comment:
You shouldn't use update:true here.  This would allow a remote user to update the value of the counter.  This is a read-only quantity.